src/patterns/organisms/footer/footer-min.hbs

```diff
@@ -0,0 +1,31 @@
+---
+name: Flexable footer component
+order: 1
+description: |
+  A page footer organism, containing common links to compliment primary navigation.
```

Comment:
sp: "complement" (technically a homonym and not a misspelling... complement with an e means they go together to complete a perfect whole, compliment with an i means the footer told the main nav it looks really nice today.)